infrasim/model/elements/pcie_port.py

```diff
@@ -40,6 +40,13 @@ def precheck(self):
         if not set(['id', 'bus', 'chassis', 'slot']).issubset(self.__port_info):
             raise ArgsNotCorrect("port \
                  <id>/<bus>/<chassis>/<slot> are all required.")
+        if 'addr' in self.__port_info:
+            if ('sec_bus' in self.__port_info) ^ ('pri_bus' in self.__port_info):
```

Comment:
why '^' here? looks a bit weird, 'and' better?

Reply:
sec_bus and pri_bus should appear together or not appear together
1  1  ok
0  1  no ok
1  0  no ok
0  0  ok